src/common/reducers/snaps.js

```diff
@@ -0,0 +1,41 @@
+import * as ActionTypes from '../actions/snaps';
+import { parseGitHubRepoUrl } from '../helpers/github-url';
+
+export function snaps(state = {
+  isFetching: false,
+  success: false,
+  error: null,
+  snaps: null,
```

Comment:
Wasn't sure if we should be naming it `snaps` or maybe `enabledRepositories`.
Technically they are snaps with some repo info on them, and we will need more info about them as snaps (builds status, name registering, snapcraft.yaml), so I guess calling them snaps is not that confusing.